src/ManageCourses.Api/Model/UserPayload.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections.Generic;
+using GovUk.Education.ManageCourses.Domain.Models;
+
+namespace GovUk.Education.ManageCourses.Api.Model
+{
+    public class ReferenceDataPayload
```

Comment:
Can you rename the file to be the name of the class

Reply:
oops thx